internal/pkg/addon/addons.go

```diff
@@ -73,8 +77,16 @@ func NewPackager(wlName string, bucket string, uploader uploader) (*Addons, erro
 		return nil, err
 	}
 
+	addons.wsPath, err = addons.ws.Path()
```

Comment:
should we move these initializations to `New` instead? after addressing Wanxian's refactor this constructor will be gone

Reply:
moving it causes all of the `deploy/cloudformation/stack` integ tests to fail - they're not configured to "be" in a copilot workspace. i was just planning on leaving it until the refactor and we'll call `Path()` when we call `Package()`.